consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java

```diff
@@ -169,12 +181,12 @@ private void actionReceivedProposal(final SignedData<ProposalPayload> msg) {
   }
 
   public void handlePrepareMessage(final SignedData<PreparePayload> msg) {
-    LOG.info("Received a prepare message.");
+    LOG.info("Received a prepare message. round={}", roundState.getRoundIdentifier());
```

Comment:
trace

Reply:
not sure on these. might belong at debug level. is this and the handle logs useful for debugging or is a bit over the top?